www/front_src/src/helpers/dynamicImport.js

```diff
@@ -1,38 +1,78 @@
 /* eslint-disable consistent-return */
 /* eslint-disable no-unused-vars */
 /* eslint-disable import/extensions */
+/* eslint-disable @typescript-eslint/explicit-function-return-type */
 
 import '../../../../node_modules/systemjs/dist/s.js'; // IIFE format so it's imported on window
 import '../../../../node_modules/systemjs/dist/extras/use-default.js'; // avoid to check module.default.default
 import './extras/global.js'; // fork global.js from systemjs to embed patch for IE (https://github.com/systemjs/systemjs/pull/2035)
 import systemCss from 'systemjs-plugin-css'; // used to import css in <head>
 
+const importModule = ({ basename, file }) => {
+  return new Promise(async (resolve, reject) => {
+    try {
+      const fileVector = `$centreonExternalModule$${file
+        .replace(/(^\.?\/)|(\.js)/g, '')
+        .replace(/\//g, '$')}`;
+      // Check if current chunk is not imported
```

Comment:
We have a lot of comments in here, I suggest that we create dedicated function with explicit name that serve as documentation instead